docs/setup_installation/common/arrow_flight_duckdb.md

```diff
@@ -0,0 +1,51 @@
+# ArrowFlight Server with DuckDB
+By default, Hopsworks uses big data technologies (Spark or Hive) to create training data and read data for Python clients.
+This is great for large datasets, but for small or moderately sized datasets (think of the size of data that would fit in a Pandas
+DataFrame in your local Python environment), the overhead of starting a Spark or Hive job and doing distributed data processing can be significant.
+
+ArrowFlight Server with DuckDB significantly reduces the time that Python clients need to read feature groups 
+and batch inference data from the Feature Store, as well as creating moderately-sized training datasets.
+
+When the service is enabled, clients will automatically use it for the following operations:
+
+- reading Feature Groups
+- reading External Feature Groups (supported Storage Connectors: Snowflake, BigQuery)
+- reading Queries (containing Feature Groups and supported External Feature Groups)
+- reading Batch Inference Data
+- creating In-Memory Training Datasets
+- writing Training Datasets
+- reading Training Datasets
```

Comment:
This is WIP right?

Reply:
writing training datasets is WIP (probably 3.4.0), reading works

Reply:
Ok but then let's remove the writing part for the moment.